### PR TITLE
Added --no-fixtures option to test baking.

### PR DESCRIPTION
**Pull request by makallio85**
I have controller, that allows post or get requests only. I was not able to bypass controller instantiating https://github.com/cakephp/bake/blob/master/src/Shell/Task/TestTask.php#L307. 

Solution was to add --no-fixtures parameter.

### CODE REVIEW

**Comment by makallio85** [comment]
Is there something I could do for this pr?

File: src/Shell/Task/TestTask.php [hunk 1]
```diff
@@ -212,15 +212,17 @@ public function bake($type, $className)
...
-        if (!empty($this->params['fixtures'])) {
-            $fixtures = array_map('trim', explode(',', $this->params['fixtures']));
-            $this->_fixtures = array_filter($fixtures);
-        } elseif ($this->typeCanDetectFixtures($type) && class_exists($fullClassName)) {
-            $this->out('Bake is detecting possible fixtures...');
-            $testSubject = $this->buildTestSubject($type, $fullClassName);
-            $this->generateFixtureList($testSubject);
-        }
+		
+		if (empty($this->params['no-fixtures'])) {
```
Comment (antograssiot) [inline src/Shell/Task/TestTask.php]: indentation seems off here, please use spaces.

**Comment by antograssiot** [comment]
@makallio85  The flag to avoid to bake fixture when baking a model is ``no-fixture`` instead of ``no-fixtures``.
We should be using the same name, it would at the same time solve the issue where when using something like ``bin/cake bake model Users --no-fixture`` will not bake fixtures files but will include the $fixture array in the ``UsersTableTest.php``.

Finally, it would be awesome if you could rebase your changes on top of master so the test suite will work properly

**Comment by makallio85** [comment]
Ok, Ill fix this next week.

**Pull request closed without merging**